ui/README.adoc

```diff
@@ -0,0 +1,17 @@
+= Mobile Security Service UI
+
+== Local Development using VSCode Chrome Debugger
+
+* Run:
+[source,shell]
+----
+npm start
+----
+to run the app in the development mode on localhost:3000
+
+* Open the project in VSCode
+* Run in debug mode (config is provided)
```

Comment:
Wdyt it be a section as `=== Debuging the project in VSCode`?

Reply:
I tried it with the heading but it looks a little funny without any content inside of it. I updated it to be an ordered list.